.github/workflows/docker.yml

```diff
@@ -17,20 +17,36 @@ jobs:
         run: docker run doctr-tf-py3.8-slim python -c 'import doctr'
 
   pytest-api:
-    runs-on: ubuntu-latest
-    env:
-        # Port for web container
-        # See docker-compose.yml
-        PORT: 8002
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python: [3.8]
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+          architecture: x64
+      - name: Install poetry
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: 1.1.13
+      - name: Lock the requirements
+        run: |
+          cd api
+          poetry lock
+          poetry export -f requirements.txt --without-hashes --output requirements.txt
+          poetry export -f requirements.txt --without-hashes --dev --output requirements-dev.txt
```

Comment:
If possible, use `make lock` so it will also "test" that Makefile works and is sync with the CI

Reply:
good catch :+1: